controller/engine_image_controller.go

```diff
@@ -590,7 +593,7 @@ func (ic *EngineImageController) cleanupExpiredEngineImage(ei *longhorn.EngineIm
 			return nil
 		}
 
-		logrus.Infof("Engine image %v (%v) expired, clean it up", ei.Name, ei.Spec.Image)
+		ic.logger.Infof("Engine image %v (%v) expired, clean it up", ei.Name, ei.Spec.Image)
```

Comment:
Consider implementing a `getLoggerForEngineImage()` function like in the below example PR.
https://github.com/longhorn/longhorn-manager/pull/747/files

We want to use fields for structured logging, so we don't want to formatF the values where it makes sense to use fields.